animation/abilities/EggplantAbility.java

```diff
@@ -0,0 +1,53 @@
+package animation.abilities;
+
+import animation.Animation;
+import characterEntities.Entity;
+import characterEntities.HitDetectionHelper;
+
+import java.awt.*;
+
+public class EggplantAbility extends Ability {
+
+	private final int ATTACK_RANGE = 75;
+
+	public EggplantAbility(Entity entity) {
+		super(entity, 2, Entity.EntityAbility.DEFAULT);
+		canDamageAnimation = new Animation(entity.getPosX(), entity.getPosY(), 75, 0, FILEPATH_ABILITY+"heroDefault", 4, 1);
```

Comment:
the abilityName is specific to the name of the icon for the bottom bar. The filepath string is, in essence different from the ability name. I would rather have a constant string